usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -37,8 +37,53 @@ blacklist {
     modprobe dm-multipath >&2
     multipath >&2
```

Comment:
IMHO you can move the `while ! multipath` up here. Only side effect is printing the "Failed to activate ..." message for every cycle of the while loop which I find totally acceptable.